ch6/boggle.py

```diff
@@ -0,0 +1,47 @@
+board = []
+dx = [-1, -1, -1, 1, 1, 1, 0, 0]
+dy = [-1, 0, 1, -1, 0, 1, -1, 1]
+
+def inRange(y, x):
```

Comment:
isRange -> is_range

Java = camelCase. e.g. isFound?
python = under_score. e.g. is_found?

this convention is used for both function and variable names